packit_service/worker/handlers/abstract.py

```diff
@@ -114,9 +114,9 @@ def _clean_workplace(self):
 
     def pre_check(self) -> bool:
         """
-        Validate the input.
+        Validate the input and skip the run if this returns False.
```

Comment:
Does documenting this here make sense? The skip itself happens in worker.jobs.process_message() if I understand it correctly. So it's up to the caller of this method to decide what to do with the return value.

Reply:
but "Validate the input" is very sparse and could mean anything, so I requested some more context for this method

Reply:
Do you want the same message as we have for events?
```
        Implement this method for those events, where you want to check if event properties are
        correct. If this method returns false during runtime, execution of service code is skipped.
```

> So it's up to the caller of this method to decide what to do with the return value.

Exactly.

Reply:
Aha! I see now! Yes, a message similar to the one events have would be much clearer.

Reply:
I've used the message from events. Is that ok?